src/rules/at-rule-blacklist/README.md

```diff
@@ -0,0 +1,49 @@
+# at-rule-blacklist
+
+Specify a blacklist of disallowed at-rules.
+
+```scss
+      @keyframes name {
+/**   ↑
+ *    Rules like this */
+```
+
+## Options
+
+`array`: `"["array", "of", "unprefixed", "at-rules"]`
+
+The rule's search is case insensitive, and it also takes vendor prefixes into account.
```

Comment:
We can remove this line as case as we don't highlight case insensitive within each rule as it's universal. Same with vendor prefixes.

For future reference, we refer to the *stylelint rule* as "This rule" to distinguish it from when a *stylelint rule* documentation needs to talk about *CSS rules*, as in [here](https://github.com/stylelint/stylelint/blob/master/src/rules/rule-non-nested-empty-line-before/README.md#rule-non-nested-empty-line-before).